pyEpiabm/pyEpiabm/tests/test_func/helper_func.py

```diff
@@ -23,23 +41,23 @@ def compare_susceptible_groups(self, small_pop, large_pop,
         ----------
         small_pop : list
             The population with fewer individuals
-        large_pop: list
+        large_pop : list
             The population with more individuals
-        status: InfectionStatus
+        status : InfectionStatus
             The infection status wants to be compared
-        method: string
+        method : string
```

Comment:
```suggestion
        method : {'greater', 'equal'}
```
not really relevant to this PR but I think this is the correct docstring when there are a few choices it should match